src/screens/BallotEjectScreen.test.tsx

```diff
@@ -55,11 +57,9 @@ test('says the ballot sheet is overvoted if it is', async () => {
         metadata: {
           ballotStyleId: '1',
           precinctId: '1',
-          ballotType: BallotType.Standard,
-          electionHash: '',
-          isTestMode: false,
-          locales: { primary: 'en-US' },
+          isTestBallot: false,
           pageNumber: 1,
+          pageCount: 2,
```

Comment:
is this some old code that snuck in there? I think that's the older data model.

Reply:
Yes. My bad. Pushing a fix.

Reply:
I hadn't run `yarn install` on `bsd`.